index.html

```diff
@@ -136,6 +136,164 @@ <h2>
+        position value</dfn> [=input value=] which is the [=logical axis
+        value=] when the [=axis=] is centered.
+      </p>
+      <p>
+        A [=gamepad=] may have [=button=] inputs. A <dfn>button</dfn> is an
+        input control that can be pressed to activate. A [=gamepad=] has a
+        <dfn>button list</dfn> which is a [=list=] containing all the
+        [=button=] inputs for the [=gamepad=] in some order determined by the
+        [=user agent=].
+      </p>
+      <p>
+        An input control may be designed to automatically return a [=button=]
+        to an unpressed state when the user stops interacting with the input
+        control. If so, the [=button=] has a <dfn>preferred button state</dfn>.
+      </p>
+      <p>
+        A [=button=] may have a digital switch to indicate when the [=button=]
+        is activated. If so, the [=button=] has a <dfn>digital button
+        value</dfn> [=input value=] that is `true` when the [=button=] is
+        activated and `false` otherwise.
+      </p>
+      <p>
+        A [=button=] may have an analog sensor enabling the [=button=] to
+        report the degree to which the [=button=] is activated. If so, the
+        [=button=] has these [=input values=]:
+      </p>
+      <ul>
+        <li>An <dfn>analog button value</dfn>, an integer value that represents
+        the degree of activation.
+        </li>
+        <li>A <dfn>minimum analog value</dfn> representing no activation. The
+        smallest in-bounds [=analog button value=].
+        </li>
+        <li>A <dfn>maximum analog value</dfn> representing full activation. The
+        largest in-bounds [=analog button value=].
+        </li>
+      </ul>
+      <p>
+        A [=button=] may be capable of detecting touch. If so, the [=button=]
+        has a <dfn>button touch value</dfn> [=input value=] that is `true` when
+        the [=button=] is touched and `false` otherwise.
+      </p>
+      <p>
+        A [=gamepad=] may have [=touch surfaces=]. A <dfn>touch surface</dfn>
+        is an input control that provides 2D position data representing points
+        of contact. A [=gamepad=] has a <dfn>touch surface list</dfn> which is
+        a [=list=] containing the [=touch surfaces=] for the [=gamepad=]. The
+        list is ordered such that [=touch surfaces=] closer to the left side of
+        the [=gamepad=] appear closer to the start of the list.
+      </p>
+      <p>
+        A [=touch surface=] has an <dfn>active touch point list</dfn> [=input
+        value=], a [=list=] of zero or more [=touch points=] representing the
+        points of contact currently detected by the sensor, ordered such that
+        earlier [=touch points=] appear closer to the start of the [=list=]. A
+        <dfn>touch point</dfn> represents a single point of contact at a single
+        point in time. A [=touch point=] has a <dfn>touch x coordinate</dfn>
+        representing its position along the user's left-right axis and a
```

Comment:
Maybe "horizontal" is more formal?
```suggestion
        representing its position along the user's horizontal axis and a
```

Reply:
When we describe the thumbstick axes we use both Horizontal and Left/Right, let's do the same here to minimize ambiguity.

I think there's another issue here, aligning to the horizontal axis only makes sense if the touchpad is on the top/bottom/front/back of the gamepad. If it's on the left or right side then the left/right axis is perpendicular to the surface of the touchpad. To make this generic we should define the coordinate system for any orientation.